requirements.txt

```diff
@@ -10,5 +10,5 @@ graphviz>=0.13
 seaborn>=0.11.1
 category_encoders>=2.0.0
 statsmodels >= 0.12.2
-imbalanced-learn>=0.7.0
+imbalanced-learn>=0.8.0
```

Comment:
Why do we need `0.8.0` now?

Reply:
`0.8.0` includes `SMOTEN`, which we use when the dataset is entirely categorical

Reply:
Sweet, thanks for explaining!